tests/test_sphere.py

```diff
@@ -0,0 +1,82 @@
+import unittest
```

Comment:
Please move these tests to `tests/test_asarray.py` (this file already exists in the current master branch).